apps/web-giddh/src/app/vouchers/list/list.component.ts

```
@@ -2473,4 +2578,534 @@ export class VoucherListComponent implements OnInit, OnDestroy {
+
+    /**
+     * Auto-fills the GST number field for E-invoice
+     *
+     * @private
+     * @memberof VoucherListComponent
+     */
+    private fetchCompanyGstDetails(): void {
+        let branches = [];
+        let currentBranch;
+        this.componentStore.branchList$.pipe(take(1)).subscribe(response => {
+            if (response && response.length) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)